doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -535,24 +543,125 @@ fulfill plant nitrogen demands.
-stem C and N into the litter pool is in contrast to the approach for unmanaged PFTs which
-puts live stem C and N into dead stem pools first. Leaf and root C and N pools
-are routed to the litter pools in the same manner as natural vegetation. Whereas food C and N
-was formerly transferred to the litter pool, CLM5 routes food C and N
+Variables track the flow of grain C and N to food and of all other plant pools, including live stem C and N, to litter, and to biofuel feedstock.
+A fraction (determined by :math:`biofuel\_harvfrac`) of leaf/livestem C and N from bioenergy crops is removed at harvest for biofuels 
+(Equations :eq:`25.9`, :eq:`25.10`, :eq:`25.12`, and :eq:`25.13`),
+with the remaining portions go to the litter pools (Equations :eq:`20.14)`, :eq:`25.11`, and :eq:`25.14`).
+Putting live stem C and N into the litter and biofuel pools is in contrast to the approach for unmanaged PFTs which
+puts live stem C and N into dead stem pools first. 
+Leaf C and N pools are routed to the little and biofuel pools, contrast to that of unmanaged PFTs which put leaf C and N to litter pools only.
+Root C and N pools are routed to the litter pools in the same manner as natural vegetation.
+  
+.. math::
+   :label: 25.9
+
+     CF_{leaf,biofuel} = \left({CS_{leaf} \mathord{\left/ {\vphantom {CS_{leaf}  \Delta t}} \right. \kern-\nulldelimiterspace} \Delta t} 
+     \right) * biofuel\_harvfrac
+     
+.. math::
+   :label: 25.10
+
+     CF_{livestem,biofuel} = \left({CS_{livestem} \mathord{\left/ {\vphantom {CS_{leaf}  \Delta t}} \right. \kern-\nulldelimiterspace} \Delta t} 
+     \right) * biofuel\_harvfrac 
+     
+.. math::
+   :label: 25.11
+
+     CF_{livestem,litter} = \left({CS_{livestem} \mathord{\left/ {\vphantom {CS_{livestem}  \Delta t}} \right. \kern-\nulldelimiterspace} \Delta t} 
+     \right) * \left( 1-biofuel\_harvfrac  \right) +CF_{alloc,livestem}
+
+with corresponding nitrogen fluxes:
+
+.. math::
+   :label: 25.12
+
+     NF_{leaf,biofuel} = \left({NS_{leaf} \mathord{\left/ {\vphantom {NS_{leaf}  \Delta t}} \right. \kern-\nulldelimiterspace} \Delta t} 
+     \right) * biofuel\_harvfrac
+     
+.. math::
+   :label: 25.13
+
+     NF_{livestem,biofuel} = \left({NS_{livestem} \mathord{\left/ {\vphantom {NS_{livestem}  \Delta t}} \right. \kern-\nulldelimiterspace} \Delta t} 
+     \right) *  biofuel\_harvfrac
+     
+.. math::
+   :label: 25.14
+
+     NF_{livestem,litter} = \left({NS_{livestem} \mathord{\left/ {\vphantom {NS_{livestem}  \Delta t}} \right. \kern-\nulldelimiterspace} \Delta t} 
+     \right) *  \left( 1-biofuel\_harvfrac  \right)
+
+where :math:`biofuel\_harvfrac` is the harvested fraction of leaf/livestem for biofuel feedstocks.
+
+.. _Table Plant functional type (PFT) parameters for harvested fraction of leaf/livestem for bioenergy crops:
+
+.. table:: Plant functional type (PFT) parameters for harvested fraction of leaf/livestem for bioenergy crops.
```

Comment:
You include all active PFTs here, so I suggest changing the description slightly, changing the word “crops” to “production”.